src/components/simple-table/__stories__/SimpleTable.stories.tsx

```diff
@@ -243,6 +243,9 @@ export const Selectable: StoryObj<SimpleTableProps<object>> = {
         items={items}
         keyExtractor={item => String(item.id)}
         loading={loading}
+        onRowClick={() => {
+          alert(`Should not see this on checkbox click`);
+        }}
```

Comment:
I think it doesn't trigger the onRowClick at all and it doesn't matter where you click on the row 🤔